From 081101c3d609a36128e696facb85d13b88a14f3e Mon Sep 17 00:00:00 2001
From: Jesper Juhl <jj@chaosbits.net>
Date: Fri, 24 Jun 2011 02:40:08 -0700
Subject: [PATCH 2/3] net: Remove unneeded version.h includes from drivers/net/bna
Git-commit: 95cb3656c09fcf6577138337216c27744ebc91e2
Patch-mainline: v3.1-rc1
References: bnc#689230, fate#311451

It was pointed out by 'make versioncheck' that some includes of
linux/version.h are not needed in drivers/net/.
This patch removes them.

Signed-off-by: Jesper Juhl <jj@chaosbits.net>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Michal Marek <mmarek@suse.cz>

---
 drivers/net/bna/cna.h |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/drivers/net/bna/cna.h b/drivers/net/bna/cna.h
index 13b9769..4eaeb46 100644
--- a/drivers/net/bna/cna.h
+++ b/drivers/net/bna/cna.h
@@ -19,7 +19,6 @@
 #ifndef __CNA_H__
 #define __CNA_H__
 
-#include <linux/version.h>
 #include <linux/kernel.h>
 #include <linux/types.h>
 #include <linux/mutex.h>
-- 
1.7.1

